pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -140,6 +140,7 @@ var _ = Describe("Generating fluentd config", func() {
 				@type tail
 				@id container-input
 				path "/var/log/containers/*.log"
+				exclude_path ['fluentd-*_openshift-logging_fluentd-*.log']
```

Comment:
nit, but is it necessary to have the container name in there too?
e.g. `'fluentd-*_openshift-logging_*.log'`

Reply:
Corrected.